README.md

```diff
@@ -1,13 +1,23 @@
-# Happy Thoughts
+# Happy Thoughts :sparkling_heart:
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was to make a kind of Twitter App in React by fetching and posting to an API. The user can post and like happy thoughts.
```

Comment:
Anna, I really enjoyed reviewing your code - as well as visiting your page. Great job that you pushed the limits and implemented much more features than blue level. Very impressive 👏 🤩 

Even your code were clean and easy to follow. I learnt a lot reviewing your code. 
You should be proud of yourself!